protocol/messenger_builder_test.go

```diff
@@ -83,6 +87,7 @@ func newTestMessenger(waku types.Waku, config testMessengerConfig) (*Messenger,
 		WithToplevelDatabaseMigrations(),
 		WithBrowserDatabase(nil),
 		WithCuratedCommunitiesUpdateLoop(false),
+		WithAccountsFeed(accountsFeed),
```

Comment:
Do we need this in all tests? Maybe only pass it as `testMessengerConfig.extraOptions` when needed?

Reply:
Yeah, It's not required for all the tests. I will clean this up 👍